introduction.md

```diff
@@ -6,33 +6,31 @@ layout: documentation
 
 # About openHAB
 
-openHAB is **software** for integrating different home automation systems and technologies into one single solution that allows over-arching automation rules, and that offers uniform user interfaces.
+openHAB is **software** for integrating different home automation systems, devices and technologies into one single solution.
+It provides uniform user interfaces, and a common approach to automation rules across the entire system, regardless of the number of manufacturers and sub-systems involved.
 
-But openHAB is also a **community** of users, contributors and maintainers.
+openHAB is also a **community** of users, contributors and maintainers, working together on an open-source, interoperable approach to home automation.
 
 ## openHAB Structure
 
-openHAB 2 is an open-source solution based on the [Eclipse SmartHome](https://eclipse.org/smarthome/) framework. It is written in Java and uses [Apache Karaf](http://karaf.apache.org/) together with [Eclipse Equinox](https://www.eclipse.org/equinox/) as an OSGi runtime and bundles this with [Jetty](https://www.eclipse.org/jetty/) as an HTTP server.
+openHAB 2 is based on the [Eclipse SmartHome](https://eclipse.org/smarthome/) framework, and is written in Java.
+It uses [Apache Karaf](http://karaf.apache.org/) together with [Eclipse Equinox](https://www.eclipse.org/equinox/) to create an Open Services Gateway initiative (OSGi) runtime environment.
+[Jetty](https://www.eclipse.org/jetty/) is used as an HTTP server.
 
-openHAB is highly modular software, which means that the base installation (the "runtime") can be extended through different kinds of "add-ons", either to communicate with new home automation solution, or to offer a new kind of user interface.
+openHAB is highly modular software, and can be extended through "add-ons".
```

Comment:
Maybe better `Add-ons` to keep it equal to the image?

Reply:
Agree, but question - what do you mean, "to keep it equal to the image"?

Reply:
The image has those 3 Add-on boxes with the same spelling.
That was meant with the "equal to".

Reply:
This comment has been addressed.